This locale can be generated only after localedef is fixed, see BZ870.

Index: glibc-2.3.5/localedata/SUPPORTED
===================================================================
--- glibc-2.3.5.orig/localedata/SUPPORTED
+++ glibc-2.3.5/localedata/SUPPORTED
@@ -83,7 +83,6 @@
 de_LU.UTF-8/UTF-8 \
 de_LU/ISO-8859-1 \
 de_LU@euro/ISO-8859-15 \
-dz_BT/UTF-8 \
 el_GR.UTF-8/UTF-8 \
 el_GR/ISO-8859-7 \
 en_AU.UTF-8/UTF-8 \
